client/js/components/procedure/basicSettings/AutoSwitchProcedurePhaseForm.vue

```diff
@@ -119,6 +123,16 @@ export default {
       default: () => ({})
     },
 
+    dataCyEndDate: {
+      type: String,
+      required: false
+    },
+
+    dataCyStartDate: {
+      type: String,
+      required: false
+    },
```

Comment:
for not required fields , we always should use a default

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/1860/commits/3599e2edea3302d1b997804176e852f77f2b4ef6